tests/cross/src/test/scala/tests/hover/HoverScala3TypeSuite.scala

```diff
@@ -71,6 +71,21 @@ class HoverScala3TypeSuite extends BaseHoverSuite {
        |""".stripMargin.hover
   )
 
+  // https://github.com/scalameta/metals/issues/1991
+  check(
+    "extension-methods",
```

Comment:
This one was fixed in dotty?

Reply:
Might be fixed in 3.0.0-RC2, so I wanted to add the test case to close the issue when updating.